linalg/densemat.cpp

```diff
@@ -1164,7 +1164,7 @@ void DenseMatrix::Eigensystem(DenseMatrix &b, Vector &ev,
    MFEM_CONTRACT_VAR(b);
    MFEM_CONTRACT_VAR(ev);
    MFEM_CONTRACT_VAR(evect);
-   mfem_error("DenseMatrix::Eigensystem for generalized eigenvalues");
+   mfem_error("DenseMatrix::Eigensystem: Compiled without LAPACK");
```

Comment:
It would be best to distinguish this message from that on line 1086.  Try to incorporate the fact that this is for the generalized eigenvalue case.

Reply:
I agree but adding to much information will exceed the character limit for the line. I can fit "(generalized)" in there otherwise we'll need to split the line.